onnx/runtime/aionnx/op_average_pool.py

```diff
@@ -0,0 +1,252 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=W0221,R0913,R0914
+
+import itertools
+from typing import Optional, Tuple
+
+import numpy  # type: ignore
```

Comment:
`import numpy as np` to follow its convention?

Reply:
I'll do that at the end. I used many lines of codes I wrote a while ago. I usually use numpy for my teachings because it is less confusing for students.